scripts/tft/app.py

```diff
@@ -0,0 +1,376 @@
+import asyncio
+import contextlib
+import json
+import pprint
+import re
+import sys
+
+from asyncio import FIRST_EXCEPTION
+from typing import AsyncContextManager, List, Tuple
+
+import aiohttp
+import git
+import tmt
+import yarl
+
+from envparse import env
+from giturlparse import parse as parse_git_url
+from loguru import logger
+
+from scripts.tft.models import Artifact, ArtifactType, CPUArch, Environment, Fmf, Os, Test, TFTRequest
+
+
+assert sys.version_info > (3, 7), "Script can't be run with lower version of python"
+
+
+# Mapping of tmt plans origin_vm_name provisioner definition to tft composes
+VM2COMPOSES = {
+    "c2r_centos7_template": "CentOS-7",
+    "c2r_centos8_template": "CentOS-8",
+    "c2r_oracle7_template": "Oracle-Linux-7.9",
+    "c2r_oracle8_template": "Oracle-Linux-8.4",
```

Comment:
```suggestion
    "c2r_centos7_template": "CentOS-7-latest",
    "c2r_centos8_template": "CentOS-8-latest",
    "c2r_oracle7_template": "Oracle-Linux-7.9",
    "c2r_oracle8_template": "Oracle-Linux-8.4",
```
The internal TFT (which we use) has different compose names than the public TFT.